android/assets/jsons/translations/template.properties

```diff
@@ -34,6 +34,17 @@ See your stats breakdown!\nEnter the Overview screen (top right corner) >\nClick
 Oh no! It looks like something went DISASTROUSLY wrong! This is ABSOLUTELY not supposed to happen! Please send me (yairm210@hotmail.com) an email with the game information (menu -> save game -> copy game info -> paste into email) and I'll try to fix it as fast as I can! = 
 Oh no! It looks like something went DISASTROUSLY wrong! This is ABSOLUTELY not supposed to happen! Please send us an report and we'll try to fix it as fast as we can! = 
 
+# Crash screen
+
+An unrecoverable error has occurred in Unciv: = 
+If this keeps happening, you can try disabling mods. = 
+You can also report this on the issue tracker. = 
+Copy = 
```

Comment:
"Copy to clipboard" already exists, we can use that

Reply:
I think "Copy to clipboard" is too long? The current buttons have a nice combined length IMO; See the screenshot.

Reply:
K fair enough

Reply:
FWIW I think there's a semantic/applicability difference too… "Copy" when it's visually obvious that there's a text field somewhere, "Copy to clipboard" when the user doesn't think of the current UI context as displaying text (E.G. save files).

I would also want to use "Copy" in at least one other place in my scripting API branch (if I ever get that done), for example.